web/api/web_api_v1.c

```diff
@@ -1191,6 +1191,50 @@ int web_client_api_request_v1_anomaly_event_info(RRDHOST *host, struct web_clien
     freez(s);
     return HTTP_RESP_OK;
 }
+
+int web_client_api_request_v1_anomaly_rate_info(RRDHOST *host, struct web_client *w, char *url) {
+    if (!netdata_ready)
+        return HTTP_RESP_BACKEND_FETCH_FAILED;
+
+    uint32_t after = 0, before = 0;
+
+    while (url) {
+        char *value = mystrsep(&url, "&");
+        if (!value || !*value)
+            continue;
+
+        char *name = mystrsep(&value, "=");
+        if (!name || !*name)
+            continue;
+        if (!value || !*value)
+            continue;
+
+        if (!strcmp(name, "after"))
+            after = (uint32_t) (strtoul(value, NULL, 0) / 1000);
+        else if (!strcmp(name, "before"))
+            before = (uint32_t) (strtoul(value, NULL, 0) / 1000);
+    }
+
+    char *s;
+    if (!before || !after)
+        s = strdup("{\"error\": \"missing after/before parameters\" }\n");
+    else {
+        s = ml_get_anomaly_rate_info(host, after, before);
+        if (!s)
+            s = strdup("{\"error\": \"json string is empty\" }\n");
+    }
```

Comment:
Use `strdupz` instead of `strdup`, just to be consistent with the rest of the code.

Reply:
Changed.